OpenRA.Game/Renderer.cs

```diff
@@ -123,26 +137,28 @@ public void InitializeDepthBuffer(MapGrid mapGrid)
 		public void BeginFrame(int2 scroll, float zoom)
 		{
 			Context.Clear();
+			screenBuffer.Bind();
 			SetViewportParams(scroll, zoom);
 		}
 
+		Size lastFramebufferResolution;
 		public void SetViewportParams(int2 scroll, float zoom)
 		{
 			// PERF: Calling SetViewportParams on each renderer is slow. Only call it when things change.
 			var resolutionChanged = lastResolution != Resolution;
 			if (resolutionChanged)
 			{
-				lastResolution = Resolution;
-				SpriteRenderer.SetViewportParams(lastResolution, 0f, 0f, 1f, int2.Zero);
+				lastFramebufferResolution = Exts.NextPowerOf2(Resolution);
```

Comment:
Resolution.NextPowerOf2();

Reply:
Also need to replace/update the framebuffer With the new size if this changes